css/styles.less

```diff
@@ -608,29 +608,23 @@ table{
 
 //main-features ==================================================================
 
-.main-features{
-//   border-bottom: 1px solid @gray-divider; 
+.main-features {
   background: @main-bg-color;
-  padding-bottom: 40px!important;
     
-  .row>.col-md-6{
+  .row >.col-md-6 {
     margin-bottom: 20px;
   }
-    h3{
-//      color: @primary-color-dark;
```

Comment:
nit: i'd say yank the commented things altogether

Reply:
Do you mean across the whole stylesheet?

Reply:
@dimitraz I'd say just these, although maybe there's a bunch of commented stuff already all over the place?

Reply:
@jasonmadigan yep so these are already deleted anyways, if there's time I might go through and clean up the commented stuff / format the less in a separate PR

Reply:
perfect, not a big deal anyway